Unit3_Real/Unit3_Real/Controller/PeopleDetailViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PeopleDetailViewController.swift
+//  Unit3_Real
+//
+//  Created by EricM on 9/4/19.
+//  Copyright © 2019 EricM. All rights reserved.
+//
+
+import UIKit
+
+class PeopleDetailViewController: UIViewController {
+    var peopleInfo: Person!
+    
+    @IBOutlet weak var image1Outlet: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var cityLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        nameLabel.text = "\(peopleInfo.name.first) \(peopleInfo.name.last)"
```

Comment:
setup all the labels and image in a private func and then call it in the viewDidLoad instead of doing it directly in there.